pset-functions.js

```diff
@@ -147,6 +257,20 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function doubleLetters(word) {
+  let letters = word.slice('')
+  for (let i = 0; i < letters.length; i++) {
+    if (letters[i] === letters[i + 1]) {
+      return true
+    }
+    else {
```

Comment:
Really great use of a for loop and taking advantage of the iteration as the index for a string. You can avoid writing the else block here (it's not required).